memtier_benchmark.cpp

```diff
@@ -790,6 +791,17 @@ struct cg_thread {
         return pthread_create(&m_thread, NULL, cg_thread_start, (void *)this);        
     }
 
+    int start(unsigned int cpu)
```

Comment:
in case this is not split . it should be able to receive list of cpus

Reply:
done